cmd/crictl/image.go

```diff
@@ -164,6 +171,27 @@ var listImageCommand = &cli.Command{
 			}
 		}
 		for _, image := range r.Images {
+			// don't show intermediate images if showall is not set
+			if !showAll {
+				status, err := ImageStatus(imageClient, image.Id, true)
```

Comment:
I'm new to the codebase. Looking at this my first question is - will parallelization of these requests give any benefits  and make command run faster on big number of images?